stable/oauth2-proxy/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and .Values.podDisruptionBudget.enabled (gt (.Values.replicaCount | int) 1) }}
```

Comment:
Why adding logic around `greater than >1`? PDB should sitll work with `replicaCount: 1` no?

Reply:
Yes, but in this case, it would be impossible to drain a node (since it will refuse to delete the pod because there is only one available pod).

I've mimicked the nginx-ingress logic.

Reply:
Are you ok with this logic? Would you prefer to remove this and be simply able to enable/disable it?

Reply:
Sounds good, thanks for the info.